yc response

gregorycerna opened the pull request.

gregorycerna committed:
yc response

rob0rt commented:
Please add some basic whitespace checking, so it only detects `yc` when it's by itself.

aaaanderson commented:
:rt:
On Dec 2, 2015 11:47 PM, "Robert Lynch" notifications@github.com wrote:

> Please add some basic whitespace checking, so it only detects yc when
> it's by itself.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/TexasLAN/LANBot/pull/84#issuecomment-161523255.

gregorycerna committed:
Update responses.coffee

made `yc` response only trigger if it's at the start of the word (allows for both yc and ycombinator, but stops things like daycare)

gregorycerna commented:
i dun fixed it

rob0rt commented:
Now I really appreciate that

arrdem commented:
arrback bitchez
